docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -55,6 +55,16 @@ The main features of the Media Controller API include:
 
```

Comment:
Side issue.
You can see the "This feature is supported in mobile and wearable applications only." sentence at the start of this page.
However we have mediacontroller.html for tv also.
Is that mean this sentence should be removed?

Also when we check https://docs.tizen.org/application/web/guides/multimedia/overview, 
we can see several sub pages are marked as not supporting TV profile.
Is it correct?

Reply:
You are right about this. This is a legacy problem which was not noticed by me. This should be fixed. Mediacontroller module is supported since 5.0 for TV also.